init.lua

```diff
@@ -66,10 +66,11 @@ end
 
 local function set_player_attribute(player, key, value)
 	if player.get_meta then
-		if value == nil then
-			player:get_meta():set_string(key, "")
-		else
-			player:get_meta():set_string(key, tostring(value))
+		local meta = player:get_meta()
+		if meta and value == nil then
+			meta:set_string(key, "")
+		elseif meta then
+			meta:set_string(key, tostring(value))
```

Comment:
not sure what style you'd like most in this case, return immediately would probably be safest:
```suggestion
		local meta = player:get_meta()
		if not meta then return end
		if value == nil then
			meta:set_string(key, "")
		else
			meta:set_string(key, tostring(value))
```

or

```suggestion
		local meta = player:get_meta()
		if meta then
			meta:set_string(key, value == nil and "" or tostring(value))
```
just dropping few options...